src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write code here
+function sortTableBody(tBody, column) {
+  const rows = [...tBody.rows];
+
+  const data = extractData(rows).sort((rowA, rowB) => {
+    return typeof rowA[column] === 'string'
+      ? rowA[column].localeCompare(rowB[column])
+      : rowA[column] - rowB[column];
+  });
+
+  for (let i = data.length - 1; i >= 0; i--) {
+    tBody.prepend(rows.find((r) => r.cells[0].textContent === data[i][0]));
```

Comment:
The logic for finding the row to prepend using `rows.find((r) => r.cells[0].textContent === data[i][0])` assumes that the first cell's text content is unique for each row. If this is not the case, it could lead to incorrect sorting. Consider using a more reliable identifier for the rows, such as a data attribute or a unique ID.